CMakeLists.txt

```diff
@@ -470,7 +470,7 @@ endif()
 
 if(VELOX_ENABLE_GCS)
   set_source(google_cloud_cpp_storage)
-  resolve_dependency(google_cloud_cpp_storage 2.22.0 EXACT)
+  resolve_dependency(google_cloud_cpp_storage CONFIG REQUIRED)
```

Comment:
Do we really want to drop the lower bound?

Reply:
See my comment above about protobuf.

Reply:
I don't understand what the protobuf version has to do with the other versions? There must still be a minimum version we need? If there are pairings between versions  of the dependencies we need we should at the minimum document them but ideally detect and properly check them.